diff --git a/../../web/viewer.mjs b/../viewer/viewer.mjs
index 5e5ca503..490927b8 100644
--- a/../../web/viewer.mjs
+++ b/../viewer/viewer.mjs
@@ -5664,7 +5664,7 @@ const defaultOptions = new Map([["allowedGlobalEvents", {
   value: 150,
   kind: OptionKind.VIEWER
 }], ...[["sandboxBundleSrc", {
-  value: "../build/pdf.sandbox.mjs",
+  value: "./build/pdf.sandbox.mjs",
   kind: OptionKind.VIEWER
 }]], ["sidebarViewOnLoad", {
   value: -1,
@@ -5688,7 +5688,7 @@ const defaultOptions = new Map([["allowedGlobalEvents", {
   value: true,
   kind: OptionKind.API
 }], ["cMapUrl", {
-  value: "../web/cmaps/",
+  value: "../cmaps/",
   kind: OptionKind.API
 }], ["disableAutoFetch", {
   value: false,
@@ -5718,7 +5718,7 @@ const defaultOptions = new Map([["allowedGlobalEvents", {
   value: false,
   kind: OptionKind.API
 }], ["iccUrl", {
-  value: "../web/iccs/",
+  value: "../iccs/",
   kind: OptionKind.API
 }], ["isOffscreenCanvasSupported", {
   value: true,
@@ -5730,7 +5730,7 @@ const defaultOptions = new Map([["allowedGlobalEvents", {
   value: false,
   kind: OptionKind.API
 }], ["standardFontDataUrl", {
-  value: "../web/standard_fonts/",
+  value: "../standard_fonts/",
   kind: OptionKind.API
 }], ["useSystemFonts", {
   value: undefined,
@@ -5740,13 +5740,13 @@ const defaultOptions = new Map([["allowedGlobalEvents", {
   value: 1,
   kind: OptionKind.API
 }], ["wasmUrl", {
-  value: "../web/wasm/",
+  value: "../wasm/",
   kind: OptionKind.API
 }], ["workerPort", {
   value: null,
   kind: OptionKind.WORKER
 }], ["workerSrc", {
-  value: "../build/pdf.worker.mjs",
+  value: "./build/pdf.worker.mjs",
   kind: OptionKind.WORKER
 }]]);
 class AppOptions {
@@ -7589,7 +7589,7 @@ class Localization {
       if (typeof console !== "undefined") {
         const locale = bundle.locales[0];
         const ids = Array.from(missingIds).join(", ");
-        console.warn(`[fluent] Missing translations in ${locale}: ${ids}`);
+        // console.warn(`[fluent] Missing translations in ${locale}: ${ids}`);
       }
     }
     if (!hasAtLeastOneBundle && typeof console !== "undefined") {
@@ -8194,11 +8194,11 @@ class SignatureStorage {
 function initCom(app) {}
 class Preferences extends BasePreferences {
   async _writeToStorage(prefObj) {
-    localStorage.setItem("pdfjs.preferences", JSON.stringify(prefObj));
+    // localStorage.setItem("pdfjs.preferences", JSON.stringify(prefObj));
   }
   async _readFromStorage(prefObj) {
     return {
-      prefs: JSON.parse(localStorage.getItem("pdfjs.preferences"))
+      prefs: undefined // JSON.parse(localStorage.getItem("pdfjs.preferences"))
     };
   }
 }
@@ -14126,6 +14126,9 @@ function renderProgress(index, total) {
 }
 window.addEventListener("keydown", function (event) {
   if (event.keyCode === 80 && (event.ctrlKey || event.metaKey) && !event.altKey && (!event.shiftKey || window.chrome || window.opera)) {
+    if (window.parent !== window) {
+      return;
+    }
     window.print();
     event.preventDefault();
     event.stopImmediatePropagation();
@@ -19695,7 +19698,7 @@ class PDFViewer {
     this.#enableNewAltTextWhenAddingImage = options.enableNewAltTextWhenAddingImage === true;
     this.imageResourcesPath = options.imageResourcesPath || "";
     this.enablePrintAutoRotate = options.enablePrintAutoRotate || false;
-    this.removePageBorders = options.removePageBorders || false;
+    this.removePageBorders = options.removePageBorders || true;
     this.maxCanvasPixels = options.maxCanvasPixels;
     this.maxCanvasDim = options.maxCanvasDim;
     this.capCanvasAreaFactor = options.capCanvasAreaFactor;
@@ -20598,7 +20601,7 @@ class PDFViewer {
           hPadding *= 2;
         }
       } else if (this.removePageBorders) {
-        hPadding = vPadding = 0;
+        if (this._scrollMode === ScrollMode.HORIZONTAL || this._spreadMode === SpreadMode.NONE) { hPadding = vPadding = 0; } else { hPadding = 10; vPadding = 0; }
       } else if (this._scrollMode === ScrollMode.HORIZONTAL) {
         [hPadding, vPadding] = [vPadding, hPadding];
       }
@@ -20715,7 +20718,7 @@ class PDFViewer {
         let hPadding = (/* inlined export .SCROLLBAR_PADDING */40),
           vPadding = (/* inlined export .VERTICAL_PADDING */5);
         if (this.removePageBorders) {
-          hPadding = vPadding = 0;
+          if (this._scrollMode === ScrollMode.HORIZONTAL || this._spreadMode === SpreadMode.NONE) { hPadding = vPadding = 0; } else { hPadding = 10; vPadding = 0; }
         }
         widthScale = (this.container.clientWidth - hPadding) / width / PixelsPerInch.PDF_TO_CSS_UNITS;
         heightScale = (this.container.clientHeight - vPadding) / height / PixelsPerInch.PDF_TO_CSS_UNITS;
@@ -23466,10 +23469,10 @@ class ViewHistory {
   }
   async _writeToStorage() {
     const databaseStr = JSON.stringify(this.database);
-    localStorage.setItem("pdfjs.history", databaseStr);
+    // localStorage.setItem("pdfjs.history", databaseStr);
   }
   async _readFromStorage() {
-    return localStorage.getItem("pdfjs.history");
+    return // localStorage.getItem("pdfjs.history");
   }
   async set(name, val) {
     await this._initializedPromise;
@@ -23606,7 +23609,7 @@ class ViewsManager extends Sidebar {
       this.#dispatchEvent();
       return;
     }
-    this.switchView(view, true);
+    this.switchView(view, false);
     if (!this.isInitialEventDispatched) {
       this.#dispatchEvent();
     }
@@ -24503,7 +24506,7 @@ const PDFViewerApplication = {
         title = decodeURIComponent(getFilenameFromUrl(url));
       } catch {}
     }
-    this.setTitle(title || url);
+    // this.setTitle(title || url);
   },
   setTitle(title = this._title) {
     this._title = title;
@@ -24977,7 +24980,7 @@ const PDFViewerApplication = {
     this.metadata = metadata;
     this._contentDispositionFilename ??= contentDispositionFilename;
     this._contentLength ??= contentLength;
-    console.log(`PDF ${pdfDocument.fingerprints[0]} [${info.PDFFormatVersion} ` + `${(metadata?.get("pdf:producer") || info.Producer || "-").trim()} / ` + `${(metadata?.get("xmp:creatortool") || info.Creator || "-").trim()}` + `] (PDF.js: ${version || "?"} [${build || "?"}])`);
+    // console.log(`PDF ${pdfDocument.fingerprints[0]} [${info.PDFFormatVersion} ` + `${(metadata?.get("pdf:producer") || info.Producer || "-").trim()} / ` + `${(metadata?.get("xmp:creatortool") || info.Creator || "-").trim()}` + `] (PDF.js: ${version || "?"} [${build || "?"}])`);
     const pdfTitle = this._docTitle;
     if (pdfTitle) {
       this.setTitle(`${pdfTitle} - ${this._contentDispositionFilename || this._title}`);
@@ -25125,9 +25128,9 @@ const PDFViewerApplication = {
     this.viewsManager?.setInitialView(sidebarView);
     setViewerModes(scrollMode, spreadMode);
     if (this.initialBookmark) {
-      setRotation(this.initialRotation);
+      // setRotation(this.initialRotation);
       delete this.initialRotation;
-      this.pdfLinkService.setHash(this.initialBookmark);
+      // this.pdfLinkService.setHash(this.initialBookmark);
       this.initialBookmark = null;
     } else if (storedHash) {
       setRotation(rotation);
@@ -25243,8 +25246,8 @@ const PDFViewerApplication = {
     eventBus.on("presentationmodechanged", evt => pdfViewer.presentationModeState = evt.state, opts);
     eventBus.on("presentationmode", this.requestPresentationMode.bind(this), opts);
     eventBus.on("switchannotationeditormode", evt => pdfViewer.annotationEditorMode = evt, opts);
-    eventBus.on("print", this.triggerPrinting.bind(this), opts);
-    eventBus.on("download", this.downloadOrSave.bind(this), opts);
+    // eventBus._on("print", this.triggerPrinting.bind(this), opts);
+    // eventBus._on("download", this.downloadOrSave.bind(this), opts);
     eventBus.on("firstpage", () => this.page = 1, opts);
     eventBus.on("lastpage", () => this.page = this.pagesCount, opts);
     eventBus.on("nextpage", () => pdfViewer.nextPage(), opts);
@@ -25267,7 +25270,7 @@ const PDFViewerApplication = {
     eventBus.on("updatefindmatchescount", onUpdateFindMatchesCount.bind(this), opts);
     eventBus.on("updatefindcontrolstate", onUpdateFindControlState.bind(this), opts);
     eventBus.on("fileinputchange", onFileInputChange.bind(this), opts);
-    eventBus.on("openfile", onOpenFile.bind(this), opts);
+    // eventBus._on("openfile", onOpenFile.bind(this), opts);
     eventBus.on("pagesedited", this.onPagesEdited.bind(this), opts);
     eventBus.on("saveextractedpages", this.onSavePages.bind(this), opts);
     eventBus.on("saveandload", this.onSaveAndLoad.bind(this), opts);
@@ -26367,7 +26370,7 @@ function webViewerLoad() {
   try {
     parent.document.dispatchEvent(event);
   } catch (ex) {
-    console.error("webviewerloaded:", ex);
+    // console.error("webviewerloaded:", ex);
     document.dispatchEvent(event);
   }
   PDFViewerApplication.run(config);
@@ -26381,4 +26384,3 @@ if (document.readyState === "interactive" || document.readyState === "complete")
 
 export { PDFViewerApplication, AppConstants as PDFViewerApplicationConstants, AppOptions as PDFViewerApplicationOptions };
 
-//# sourceMappingURL=viewer.mjs.map
\ No newline at end of file
